data/publiccloud/azure_runcmd.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash -u
+set -o pipefail
+. $(dirname "${BASH_SOURCE[0]}")/azure_lib_fn.sh
+# Create a vm and use run-command to execute RunShellscript
+# list, show , delete the created resource
+# set -x
+
+#Required AZ setting variable block
+grpname="${1:-oqaclitest}"
+location="${2:-westus}"
+vmname="${3:-oqacliruncmdvm}"
+ssh_key="${4:-oqaclitest-sshkey}"
+vmximagename="${5:-UbuntuLTS}"
+
+#local variable block
+account="Azure RD"
+admin="azureuser"
+runcmd="oqaclirc"
+
+echo "**Start of AZURE CLI run-command test"
+echo "*************************************"
+
+# Resource creation
+# echo "Set Azure Account { az account set }"
+cmd_status "az_account_set" az account set -n "$account"
+echo "Azure account set to $account"
+
+
+echo "Creating resource group { az group create }"
+cmd_status "az_group_create" az group create -n "$grpname" -l "$location" -o table
+echo "Created group $grpname"
+
+#start vm provision
+echo "Creating vm $vmname { az vm create }"
+cmd_status "az_vm_create" az vm create \
+    --name "$vmname" \
+    --resource-group "$grpname" \
+    --location "$location" \
+    --public-ip-sku Standard \
+    --image "$vmximagename" \
+    --generate-ssh-keys \
+    --admin-username "$admin" \
+    --enable-agent true \
+    --enable-auto-update true \
```

Comment:
why do we need this ? it will cause Azure try to inject some extra logic into VM . which is additional point where test could fail . On other hand less likely such failure is something what we interested about

Reply:
Ok. this one can be removed.